python/kserve/kserve/metrics.py

```diff
@@ -0,0 +1,20 @@
+from prometheus_client import Histogram
+import os
+
+PROM_KEYS = {"K_SERVICE": "service_name", "K_CONFIGURATION": "configuration_name",
```

Comment:
how are these env vars inserted?
by kserve controller?

Reply:
for raw deployment mode, we do not have configuration and revision but we probably still need service and deployment name.

Reply:
with this code, we will still add them? or are you suggesting a change?